### PR TITLE
Hamburger page

### PR DESCRIPTION
**Pull request by Ejikeokegbe**
Added span to create hamburger button
Added JavaScript to the hamburger to make the site dynamics
Added Hello to the menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,20 +18,21 @@
...
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
+        <a href="#" class="logo">My Logo</a>
+          <ul class="nav-menu">
+            <li class="nav-content"><a href="#" class="nav-link">Hello</a></li>
+            <li class="nav-content"><a href="#sectina" class="nav-link">Portfolio</a></li>
+            <li class="nav-content"><a href="#aboutme" class="nav-link">About</a></li>
+            <li class="nav-content"><a href="#contactme" class="nav-link">Contact</a></li>
           </ul>
```
Comment (divinecharlotte) [inline index.html:27]: - [ ] Kindly consider making sure that your  hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) to make your it an exact duplicate of the given design while doing so make sure the following look exactly the same as in the given design:
- hamburger color 
- nav links color
- padding and margins
check the images below for more clarification
- 1.your design 👇 
Screen Shot 2023-05-22 at 18 06 36

- 2.Figma design 👇 
Screen Shot 2023-05-22 at 18 07 31

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team      , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- linters are passing 👍 
- Descriptive Pr 👍 




 

## Required Changes ♻️

- [ ] kindly consider adding a working link to your LinkedIn account under the authors section to give a good user experience to the people visiting your readme 

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (OyePriscilla) [reply index.html:27]: - [ ] As suggested earlier, kindly ensure that your design is close to the given template. 👍 

- [ ] Kindly take note of the following:

1. Appropriate margin (left, right and top) arrond the nav items. ℹ️ 
2. Implement the background with CSS blur properties. ℹ️ 
3. Spacing between the Portfolio and the close (X) button. ℹ️ 

## Your Design

Untitled2

## Figma Design

Untitled2

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Ejikeokegbe 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Use JavaScript to manipulate DOM elements ✔️ 
Use JavaScript events ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OyePriscilla** [comment]
@Ejikeokegbe Please kindly ensure to patiently wait after you've submitted PR for feedback, I could see that you're pushing some commits in the process of review. This is not ideal!!!

File: style.css [hunk 2]
```diff
@@ -12,23 +12,86 @@ body {
...
-  align-items: center;
-  margin-left: 40px;
-  margin-right: 40px;
-  margin-top: 15px;
-  border: none;
-}
+  .nav-link {
+    color: white;
+    text-decoration: none;
+    transition: 0.7s ease;
```
Comment (tufoinnkuo10) [inline style.css:23]: - [optional] Great job fixing the width, spacing and color like the previous reviewer requested. please also add the following so it looks like the figma version.
   ```
 font-size: 32px;
    font-weight: bolder;
```

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Ejikeokegbe 

Your project is complete! Great job for making the changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- menu works as expected
- figma design followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**